code/_onclick/hud/alert.dm

```diff
@@ -1034,39 +1018,34 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 
 // PRIVATE = only edit, use, or override these if you're editing the system as a whole
 
+/// Gets the placement for the alert based on its index
+/datum/hud/proc/get_ui_alert_placement(index)
+	// Only has support for 5 slots currently
+	if(index > 5)
+		return ""
+
+	return "EAST-1:28,CENTER+[6 - index]:[29 - index * 2]"
+
 // Re-render all alerts - also called in /datum/hud/show_hud() because it's needed there
 /datum/hud/proc/reorganize_alerts(mob/viewmob)
 	var/mob/screenmob = viewmob || mymob
 	if(!screenmob.client)
-		return
+		return FALSE
 	var/list/alerts = mymob.alerts
 	if(!hud_shown)
 		for(var/i in 1 to alerts.len)
 			screenmob.client.screen -= alerts[alerts[i]]
-		return 1
-	for(var/i in 1 to alerts.len)
+		return TRUE
+	for(var/i in 1 to length(alerts))
```

Comment:
```suggestion
	for(var/index in 1 to length(alerts))
```

i think we should move away from using the single-letter variable in this instance since this is tied to the argument of a proc we invoke

Reply:
using `i` as a loop iterator is one case which is explicitly excused in the "single letter var" guideline, especially when "index" doesn't tell much more than "i"

Reply:
yeah but i just am not fond of it since we feed it as an argument into the proc you created in this pr. likely whatever but i thought i would say something when i noticed it